ruleset/rules/0755-office365_rules.xml

```diff
@@ -0,0 +1,928 @@
+        <description>Office 365: Skype for Business events.</description>
+        <options>no_full_log</options>
+        <group>SkypeForBusinessCmdlets</group>
+    </rule>
+
+    <rule id="91554" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^24$</field>
+        <description>Office 365: Events for eDiscovery activities performed by running content searches and managing eDiscovery cases in the Security and Compliance Center.</description>
+        <options>no_full_log</options>
+        <group>Discovery</group>
+    </rule>
+
+    <rule id="91555" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^25$</field>
+        <description>Office 365: Events from Microsoft Teams.</description>
+        <options>no_full_log</options>
+        <group>MicrosoftTeams</group>
+    </rule>
+
+    <rule id="91556" level="12">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^28$</field>
+        <description>Office 365: Phishing and malware events from Exchange Online Protection and Microsoft Defender for Office 365.</description>
+        <options>no_full_log</options>
+        <group>ThreatIntelligence</group>
+    </rule>
+
+    <rule id="91557" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^29$</field>
+        <description>Office 365: Submission events from Exchange Online Protection and Microsoft Defender for Office 365.</description>
+        <options>no_full_log</options>
+        <group>MailSubmission</group>
+    </rule>
+
+    <rule id="91558" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^30$</field>
+        <description>Office 365: Microsoft Power Automate (formerly called Microsoft Flow) events.</description>
+        <options>no_full_log</options>
+        <group>MicrosoftFlow</group>
+    </rule>
+
+    <rule id="91559" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^31$</field>
+        <description>Office 365: Advanced eDiscovery events.</description>
+        <options>no_full_log</options>
+        <group>AeD</group>
+    </rule>
+
+    <rule id="91560" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^32$</field>
+        <description>Office 365: Microsoft Stream events.</description>
+        <options>no_full_log</options>
+        <group>MicrosoftStream</group>
+    </rule>
+
+    <rule id="91561" level="3">
+        <if_sid>91532</if_sid>
+        <field name="office_365.RecordType" type="osregex">^33$</field>
+        <description>Office 365: Events related to DLP classification in SharePoint.</description>
+        <options>no_full_log</options>
+        <group>ComplianceDLPSharePointClassification</group>
+    </rule>
+
+    <rule id="91562" level="5">
```

Comment:
I think this level should be higher since these alerts come from Microsoft Defender.

Reply:
Done